release.sh

```diff
@@ -5,21 +5,27 @@ if [ $# -ne 1 ]; then
     exit 1
 fi
 
-set -xe
+set -o errexit
+set -o xtrace
 
 python3 --version
 git --version
 
 version=$1
 
-sed -i "s/__version__ = .*/__version__ = '${version}'/" */__init__.py
+if [[ "$OSTYPE" == "darwin"* ]]; then
+	sed -i "" "s/__version__ = .*/__version__ = '${version}'/" */__init__.py
```

Comment:
On MacOS you need to define the backup file extension when replacing contents with `sed`. Ref: http://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux